src/app/pipes/phase-filter.pipe.ts

```diff
@@ -8,7 +8,7 @@ import { CurrentPhase } from '../models/currentphase';
 export class PhaseFilterPipe implements PipeTransform {
 
     transform(value: Project[], q: string) {
-        if (!q) {
+        if (!q || q === '') {
```

Comment:
In javascript, empty strings are falsey, making `|| q === ''` superfluous (see [this](https://stackoverflow.com/questions/19839952/all-falsey-values-in-javascript)).

Same applies to _project-region-filter.pipe.ts_.